getExclusions.m

```diff
@@ -1,7 +1,6 @@
 % getExclusionCriteria
 function [OK] = getExclusions(tcs)
```

Comment:
Could this be a more generic function that doesn't have such a  complicated input structure dependence?

Reply:
I don't think so. It's self-contained, in that this repository is the one that makes the data structures, and it's specifically designed for the `MoveStim` protocol. It's a bit general in that it can build based on `.nev` or kilo sort outputs. If I were to bundle out just the bit that's not experiment specific, it's in Matlab as `ranksum()`